src/main/java/core/basesyntax/service/operations/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (transaction.getOperation() == FruitTransaction.Operation.BALANCE) {
+            Storage.fruits.put(transaction.getFruit(), transaction.getQuantity());

```

Comment:
Directly accessing Storage.fruits may lead to violation of encapsulation. Consider creating getter and setter methods for it.

Reply:
ok, done